catalog/plugin/catalog-plugin-security-audit/src/main/java/org/codice/alliance/catalog/plugin/security/audit/SecurityAuditPlugin.java

```diff
@@ -0,0 +1,134 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+
+package org.codice.alliance.catalog.plugin.security.audit;
```

Comment:
No, I don't think there is any specific reason why this plugin needs to be in Alliance. However, if we move it out of here, we will need to still drop a configuration for Alliance to hit all of the additional security attributes.